jenkins-master/jobs/scripts/workspace/runtests.py

```diff
@@ -5,108 +5,141 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 import argparse
+import copy
 import os
-import shutil
 import subprocess
-import sys
 
-here = os.path.dirname(os.path.abspath(__file__))
-venv_path = os.path.join(here, 'tests_venv')
+from config import config
+from jenkins import JenkinsDefaultValueAction
 
+# Purge unwanted environment variables like credentials
+ENV_VARS_TO_PURGE = [
+    'AWS_BUCKET', 'AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY',
+    'TREEHERDER_URL', 'TREEHERDER_CLIENT_ID', 'TREEHERDER_SECRET',
+]
```

Comment:
Why should it not be? I don't see even a warning in my editor which automatically checks that when saving the file.